addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -17,7 +17,7 @@ def setUpClass(cls):
         cls.company_data_2 = cls.setup_other_company()
 
         cls.uom_units = cls.env.ref('uom.product_uom_unit')
-        cls.uom_dozens = cls.env.ref('uom.product_uom_dozen')
+        cls.uom_hour = cls.env.ref('uom.product_uom_hour')
```

Comment:
Why is it replaced by `Hours` instead of the traditionnal `Pack of 6` like everywhere else?